### PR TITLE
fix: daco dot handling

### PR DESCRIPTION
**Pull request by ishche**
## Checklist:
- [x] Each of my commits contains one meaningful change
- [x] I have performed rebase of my branch on top of the development
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: server/dialect-daco/src/main/java/org/eclipse/lsp/cobol/dialects/daco/DaCoVisitor.java [hunk 1]
```diff
@@ -121,7 +123,11 @@ private void addReplacementContext(ParserRuleContext ctx) {
     String newText = context.getExtendedDocument().toString()
             .substring(ctx.start.getStartIndex(), ctx.stop.getStopIndex() + 1)
             .replaceAll("[^ \n]", CobolDialect.FILLER);
-    Range range = DialectUtils.constructRange(ctx);
+    Token start = ctx.start;
+    Token stop = ctx.stop;
+    Range range = new Range(new Position(start.getLine() - 1, start.getCharPositionInLine()),
+            new Position(stop.getLine() - 1,
+                    stop.getCharPositionInLine() + stop.getStopIndex() - stop.getStartIndex()));
```
Comment (slavek-kucera) [inline server/dialect-daco/src/main/java/org/eclipse/lsp/cobol/dialects/daco/DaCoVisitor.java:145]: Should this be the same?
https://github.com/eclipse-che4z/che-che4z-lsp-for-cobol/blob/development/server/dialect-idms/src/main/java/org/eclipse/lsp/cobol/dialects/idms/IdmsVisitor.java#L196-L199

Comment (ishche) [reply server/dialect-daco/src/main/java/org/eclipse/lsp/cobol/dialects/daco/DaCoVisitor.java:145]: It should be fine, daco grammar doesn't consume dots.

Comment (ishche) [reply server/dialect-daco/src/main/java/org/eclipse/lsp/cobol/dialects/daco/DaCoVisitor.java:145]: Oops, there are dots expected.

**Review by slavek-kucera — APPROVED** [review]
I suggest revisiting this in the context of the range inclusive/exclusive end in the extendedtextline.

**Comment by ishche** [comment]
> I suggest revisiting this in the context of the range inclusive/exclusive end in the extendedtextline.

https://github.com/eclipse-che4z/che-che4z-lsp-for-cobol/issues/2446 got an issue for it

**Pull request merged**